font/src/lib.rs

```diff
@@ -173,7 +173,10 @@ pub struct RasterizedGlyph {
 
 #[derive(Clone, Debug)]
 pub enum BitmapBuffer {
+    /// RGB pixels.
     RGB(Vec<u8>),
```

Comment:
If we comment it, we might as well say that this isn't really RGB pixels but an RGB alphamask?

Effectively, this is also premultiplied, just with on alpha.

Reply:
Maybe we should change it...

Reply:
I've changed it to `RGB alphamask.` in the last revision.